infra/lib/project/stacks/importCSV/import-service-stack.ts

```diff
@@ -70,7 +71,8 @@ export class ImportServiceStack extends Stack {
     });
 
     // Grant the Lambda function read permissions to the S3 bucket
-    bucket.grantRead(parserLambda);
+    bucket.grantReadWrite(parserLambda);
+    bucket.grantDelete(parserLambda);
```

Comment:
it is unnecessary, readWrite gives the same rights. I have describe the problem in the Teams